### PR TITLE
[dv/otp_ctrl] Add exclusions to csr automation tests

### PR DESCRIPTION
**Pull request by cindychip**
1. Add exclusions to CSR automation test
2. Fix port naming from tb
3. Add clear mem and drive pwr input

Signed-off-by: Cindy Chen <chencindy@google.com>

### CODE REVIEW

File: hw/ip/otp_ctrl/dv/otp_ctrl_sim_cfg.hjson [hunk 1]
```diff
@@ -29,7 +29,7 @@
                 "{proj_root}/hw/dv/data/common_sim_cfg.hjson",
                 // Common CIP test lists
                 "{proj_root}/hw/dv/data/tests/csr_tests.hjson",
-                "{proj_root}/hw/dv/data/tests/mem_tests.hjson",
+                // "{proj_root}/hw/dv/data/tests/mem_tests.hjson",
```
Comment (weicaiyang) [inline hw/ip/otp_ctrl/dv/otp_ctrl_sim_cfg.hjson:32]: nit: perhaps add TODO to enable this later

**Review by weicaiyang — APPROVED** [review]
LGTM

File: hw/ip/otp_ctrl/dv/otp_ctrl_sim_cfg.hjson (hunk 1 above)
Comment (cindychip) [reply hw/ip/otp_ctrl/dv/otp_ctrl_sim_cfg.hjson:32]: oh that is right thanks for the reminder, I forget to add that.

**Review by msfschaffner — APPROVED** [review]
LGTM, thanks!

**Pull request merged**